src/templates/stickyboard-simple/src/components/base/PageBase.js

```diff
@@ -169,6 +183,9 @@ class PageBase extends React.Component {
                                 <Sticker
                                     key={block.i}
                                     onChange={() => {}}
+                                    name={block.i}
+                                    description={description}
+                                    descriptionChange={this.descriptionChange}
```

Comment:
Please pass 'Name' and 'Description' from StickerObject.
`const StickerObject = StickerDict[block.i];`